src/components/RightSidebar/RightSidebar.vue

```diff
@@ -120,6 +128,8 @@ export default {
 			// Sidebar status before starting editing operation
 			sidebarOpenBeforeEditingName: '',
 			matterbridgeEnabled: loadState('talk', 'enable_matterbridge'),
+			description: 'sdfjkhsdkj fhs dfkjhsdfkjhs dfkjas dh fkjsdfhkasdjfhdk ajfhsd kjfhsdkfj hsdkfjhdsfkjsdhfkjsdhf',
```

Comment:
I prefer Lorem Ipsum :wink: 

Let's clear this before merging.

Reply:
Sure, once the backend is ready the conv descriptions will be stored in the conv store

Reply:
https://lotremipsum.com/

Reply:
I prefer https://baconipsum.com/